tests/utils/test_modeling_utils.py

```diff
@@ -460,6 +461,24 @@ def test_model_from_config_torch_dtype_str(self):
         with self.assertRaises(ValueError):
             model = AutoModel.from_pretrained(TINY_T5, torch_dtype="int64")
 
+    @slow
```

Comment:
Since we are loading the model on meta device, this test should be rather quick. Also it requires torch.  
```suggestion
    @require_torch
```